lib/ts/types.ts

```diff
@@ -40,7 +41,7 @@ export type SuperTokensConfig = {
     /*
      * List of recipes for authentication and session management.
      */
-    recipeList: CreateRecipeFunction<any, any, any, any>[];
+    recipeList: { authReact: CreateRecipeFunction<any, any, any, any>; webJS: CreateRecipeFunctionWebJS<any> }[];
```

Comment:
I'd prefer if you extracted this into a single `RecipeInitResult` type (it'll have many type params, but I guess that's OK) that each recipe can reuse.